src/aura/STG_CMP_CourseConnections/STG_CMP_CourseConnections.cmp

```diff
@@ -116,6 +115,39 @@
           <div class="slds-col slds-size--1-of-1">
             <ui:outputText value="{!$Label.c.stgHelpDefaultFacultyType}" class="slds-text-body--small" />
           </div>
+
+          <div class="slds-col slds-grid slds-m-top--medium">
+        	<div class="slds-col slds-size--1-of-2">
+                <div class="slds-col slds-grid slds-size--1-of-1">
+                    <div class="slds-col slds-size--1-of-2">
+                        <ui:outputText value="{!$Label.c.stgPopulateAffiliationOnStudentCourseConnectionTitle}" />
+                    </div>
+                </div>
+                <br></br>
+                <div class="slds-col">
+                    {!$Label.c.stgPopulateAffiliationOnStudentCourseConnectionTitle2}
+                    <br></br>
+                    <div class="slds-col">
+                    	<ui:outputText value="{!$Label.c.stgHelpPopulateAffiliationOnStudentCourseConnection}" class="slds-text-body--small" />
+                	</div>
+                </div>
+            </div>
+
+            <div class="slds-col slds-size--1-of-2">
+            	<label class="slds-checkbox">
+              		<aura:if isTrue="{!v.isView}">
+                  		<span style="margin-right: 8px">
+                      		<ui:outputCheckbox value="{!v.hierarchySettings.Populate_Affil_on_Student_Couse_Cxn__c}" class="slds-checkbox course-cons-outputcheck" />
+                  		</span>
+              		<aura:set attribute="else">
+                  		<ui:inputCheckbox value="{!v.hierarchySettings.Populate_Affil_on_Student_Couse_Cxn__c}" class="slds-checkbox course-cons-inputcheck"
```

Comment:
@kdy618 it looks like we may have missed a typo here? I see you corrected the settings field in other places, but this one still reads "couse"